# GraphRAG Inspection Viewer

This notebook loads **all files from the `inspection` folder** created for the Austrian PV Strategy pilot.

Place this notebook inside:

`C:\Users\HP-ZBOOK i7\graphrag_test\pilot_01`

The expected folder structure is:

```text
pilot_01/
├── inspection/
│   ├── 00_summary.txt
│   ├── 01_entities_all.csv
│   ├── ...
│   └── 13_community_report_review.csv
└── inspection_viewer.ipynb
```

This notebook only reads local files. It does **not** call OpenRouter or GraphRAG.


In [2]:
from pathlib import Path
import pandas as pd
from IPython.display import display

INSPECTION = Path(r"C:\Users\HP-ZBOOK i7\graphrag_test\pilot_01\inspection")

print("Inspection folder:", INSPECTION)
print("Folder exists:", INSPECTION.exists())

datasets = {}

for file in sorted(INSPECTION.glob("*.csv")):
    df = pd.read_csv(file, encoding="utf-8-sig")
    datasets[file.name] = df

print("\nImported datasets:")

for name, df in datasets.items():
    print(f"{name}: {len(df)} rows")

print("\nTotal CSV files imported:", len(datasets))

Inspection folder: C:\Users\HP-ZBOOK i7\graphrag_test\pilot_01\inspection
Folder exists: True

Imported datasets:
01_entities_all.csv: 312 rows
02_entity_type_counts.csv: 4 rows
03_entity_review.csv: 312 rows
04_relationships_all.csv: 414 rows
05_relationship_review.csv: 414 rows
06_communities_all.csv: 62 rows
07_community_reports_all.csv: 62 rows
08_documents_all.csv: 1 rows
09_text_units_all.csv: 17 rows
10_possible_duplicates.csv: 2 rows
11_top_connected_entities.csv: 30 rows
12_top_relationships.csv: 50 rows
13_community_report_review.csv: 62 rows

Total CSV files imported: 13


## 1. Setup and load every inspection file


In [23]:
from pathlib import Path
import pandas as pd
from IPython.display import display

ROOT = Path.cwd()
INSPECTION = ROOT / "inspection"

print("Notebook folder:", ROOT)
print("Inspection folder:", INSPECTION)
print("Inspection folder exists:", INSPECTION.exists())

if not INSPECTION.exists():
    raise FileNotFoundError(
        "The inspection folder was not found. "
        "Place this notebook inside pilot_01."
    )


Notebook folder: c:\Users\HP-ZBOOK i7\graphrag_test\pilot_01
Inspection folder: c:\Users\HP-ZBOOK i7\graphrag_test\pilot_01\inspection
Inspection folder exists: True


In [24]:
# Load all CSV files automatically
datasets = {}

for path in sorted(INSPECTION.glob("*.csv")):
    try:
        datasets[path.name] = pd.read_csv(path, encoding="utf-8-sig")
    except UnicodeDecodeError:
        datasets[path.name] = pd.read_csv(path)

print(f"Loaded {len(datasets)} CSV datasets.\n")

for name, df in datasets.items():
    print(f"{name:40s} {len(df):>6} rows x {len(df.columns):>3} columns")


Loaded 13 CSV datasets.

01_entities_all.csv                         312 rows x   8 columns
02_entity_type_counts.csv                     4 rows x   2 columns
03_entity_review.csv                        312 rows x  12 columns
04_relationships_all.csv                    414 rows x   8 columns
05_relationship_review.csv                  414 rows x  14 columns
06_communities_all.csv                       62 rows x  12 columns
07_community_reports_all.csv                 62 rows x  15 columns
08_documents_all.csv                          1 rows x   7 columns
09_text_units_all.csv                        17 rows x   8 columns
10_possible_duplicates.csv                    2 rows x   6 columns
11_top_connected_entities.csv                30 rows x   5 columns
12_top_relationships.csv                     50 rows x   4 columns
13_community_report_review.csv               62 rows x  18 columns


## 2. Read the summary file


In [25]:
summary_path = INSPECTION / "00_summary.txt"

if summary_path.exists():
    print(summary_path.read_text(encoding="utf-8"))
else:
    print("00_summary.txt was not found.")


GRAPH RAG FULL INSPECTION
Austrian Photovoltaic Strategy 2024

DOCUMENTS
Rows: 1
Columns: id, human_readable_id, title, text, text_unit_ids, creation_date, raw_data

TEXT_UNITS
Rows: 17
Columns: id, human_readable_id, text, n_tokens, document_id, entity_ids, relationship_ids, covariate_ids

ENTITIES
Rows: 312
Columns: id, human_readable_id, title, type, description, text_unit_ids, frequency, degree

RELATIONSHIPS
Rows: 414
Columns: id, human_readable_id, source, target, description, weight, combined_degree, text_unit_ids

COMMUNITIES
Rows: 62
Columns: id, human_readable_id, community, level, parent, children, title, entity_ids, relationship_ids, text_unit_ids, period, size

COMMUNITY_REPORTS
Rows: 62
Columns: id, human_readable_id, community, level, parent, children, title, summary, full_content, rank, rating_explanation, findings, full_content_json, period, size

INDEXING STATISTICS
Total runtime: 197.46940207481384 seconds
Number of documents: 1

ENTITY TYPE COUNTS
ORGANIZATION: 142


## 3. Helper: view any dataset by filename


In [26]:
def view(filename, rows=20):
    if filename not in datasets:
        print("Dataset not found:", filename)
        print("\nAvailable datasets:")
        for name in datasets:
            print(" -", name)
        return

    df = datasets[filename]

    print(filename)
    print(f"Rows: {len(df)}")
    print(f"Columns: {len(df.columns)}")
    print("\nColumn names:")
    print(list(df.columns))
    print()

    display(df.head(rows))


## 4. All entities


In [27]:
view("01_entities_all.csv", rows=30)


01_entities_all.csv
Rows: 312
Columns: 8

Column names:
['id', 'human_readable_id', 'title', 'type', 'description', 'text_unit_ids', 'frequency', 'degree']



,id,human_readable_id,title,type,description,text_unit_ids,frequency,degree
0,cb0a1b84-f9a3-4404-9807-afa3d1e6c3af,0,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,EVENT,"The ""Österreichische Photovoltaik-Strategie"" (...",['d7219f76b31818b7b13bf08dda81003d752e53f76608...,14,39
1,822d0b21-2432-4dfc-9d1f-665425d28e8b,1,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",ORGANIZATION,"The ""Bundesministerium für Klimaschutz, Umwelt...",['d7219f76b31818b7b13bf08dda81003d752e53f76608...,2,9
2,989f5ee5-4f8f-45bb-ae0c-fed8e87f1dbc,2,HUBERT FECHNER,PERSON,Author of the strategy document; involved in p...,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,1,2
3,1d4b9657-b1cb-4952-a417-bdb299846dd4,3,LEONORE GEWESSLER,PERSON,The Federal Minister of Austria for Climate Ac...,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,1,2
4,05c9af58-ac32-4ec6-b923-358b587e8cae,4,ÖSTERREICH,GEO,"Österreich, also known as Austria, is a Centra...",['d7219f76b31818b7b13bf08dda81003d752e53f76608...,15,82
5,00d38e7b-8bbe-4593-9fe9-df5bf6fb7427,5,WIEN,GEO,"WIEN is the German name for Vienna, the capita...",['d7219f76b31818b7b13bf08dda81003d752e53f76608...,3,8
6,37e2bb35-a727-40e3-b466-b12998e333aa,6,ERNEUERBAREN-AUSBAU-GESETZ (EAG),EVENT,"The ERNEUERBAREN-AUSBAU-GESETZ (EAG), known in...",['d7219f76b31818b7b13bf08dda81003d752e53f76608...,2,5
7,672d47ca-35bf-4a7a-819f-56940ff891e8,7,INTEGRIERTER ÖSTERREICHISCHER NETZINFRASTRUKTU...,EVENT,The Integrated Austrian Network Infrastructure...,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,1,1
8,c579a318-d4d8-48b0-a35d-59fb0ca93bf4,8,PHOTOVOLTAIK (PV),EVENT,The technological process of converting solar ...,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,1,4
9,8a7a07f7-e526-4c84-8dcc-ff6f50a8e68f,9,CAJETAN PERWEIN,PERSON,Cajetan Perwein is credited as a photographer ...,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,1,1


## 5. Entity type counts


In [28]:
view("02_entity_type_counts.csv", rows=50)


02_entity_type_counts.csv
Rows: 4
Columns: 2

Column names:
['raw_graphrag_type', 'count']



,raw_graphrag_type,count
0,ORGANIZATION,142
1,EVENT,121
2,PERSON,25
3,GEO,24


## 6. Entity review table

This is one of the most important files.

Use it later for:
- GraphRAG raw type
- suggested controlled-schema type
- canonical entity name
- keep/reject decision
- record linkage
- review notes


In [29]:
view("03_entity_review.csv", rows=30)


03_entity_review.csv
Rows: 312
Columns: 12

Column names:
['human_readable_id', 'title', 'type', 'description', 'frequency', 'degree', 'text_unit_ids', 'suggested_schema_type', 'canonical_name', 'keep_in_controlled_kg', 'record_linkage_group', 'review_notes']



,human_readable_id,title,type,description,frequency,degree,text_unit_ids,suggested_schema_type,canonical_name,keep_in_controlled_kg,record_linkage_group,review_notes
0,0,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,EVENT,"The ""Österreichische Photovoltaik-Strategie"" (...",14,39,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
1,1,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",ORGANIZATION,"The ""Bundesministerium für Klimaschutz, Umwelt...",2,9,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
2,2,HUBERT FECHNER,PERSON,Author of the strategy document; involved in p...,1,2,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
3,3,LEONORE GEWESSLER,PERSON,The Federal Minister of Austria for Climate Ac...,1,2,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
4,4,ÖSTERREICH,GEO,"Österreich, also known as Austria, is a Centra...",15,82,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
5,5,WIEN,GEO,"WIEN is the German name for Vienna, the capita...",3,8,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
6,6,ERNEUERBAREN-AUSBAU-GESETZ (EAG),EVENT,"The ERNEUERBAREN-AUSBAU-GESETZ (EAG), known in...",2,5,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
7,7,INTEGRIERTER ÖSTERREICHISCHER NETZINFRASTRUKTU...,EVENT,The Integrated Austrian Network Infrastructure...,1,1,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
8,8,PHOTOVOLTAIK (PV),EVENT,The technological process of converting solar ...,1,4,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
9,9,CAJETAN PERWEIN,PERSON,Cajetan Perwein is credited as a photographer ...,1,1,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN


## 7. All relationships


In [30]:
view("04_relationships_all.csv", rows=30)


04_relationships_all.csv
Rows: 414
Columns: 8

Column names:
['id', 'human_readable_id', 'source', 'target', 'description', 'weight', 'combined_degree', 'text_unit_ids']



,id,human_readable_id,source,target,description,weight,combined_degree,text_unit_ids
0,ee487943-e36d-4146-8e2e-b7a9a1fd3067,0,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,"The BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT,...",19.0,48,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
1,e87db637-71f3-4134-8a02-954d1e518207,1,HUBERT FECHNER,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,Hubert Fechner is the principal author of the ...,8.0,41,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
2,316e1382-2ca7-477e-9d9f-c5f6bab0b3ba,2,LEONORE GEWESSLER,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",Leonore Gewessler is the Federal Minister lead...,10.0,11,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
3,86b0f8da-0a86-48c8-990d-8fbbe165249b,3,LEONORE GEWESSLER,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,"Leonore Gewessler represents, promotes, and ov...",9.0,41,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
4,c2727a60-8f2d-4b7f-9fd0-25cd463e97c7,4,ÖSTERREICH,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,ÖSTERREICH (Austria) is both the creator and p...,67.0,121,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
5,48dadaa4-1765-4d04-bab8-87e9047bf629,5,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",HUBERT FECHNER,Hubert Fechner authored the strategy document ...,6.0,11,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
6,dbda7288-2916-410f-95d8-54283a4ce66a,6,WIEN,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",Vienna is the seat of the Ministry and the pub...,7.0,17,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
7,34e20a8c-8899-486c-afa6-f1467d515bfb,7,ERNEUERBAREN-AUSBAU-GESETZ (EAG),ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,The EAG is a backbone legal act referenced and...,9.0,44,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
8,3a9025be-b2d8-4591-ba85-1fd25a4a58d2,8,INTEGRIERTER ÖSTERREICHISCHER NETZINFRASTRUKTU...,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,The strategy references and builds upon the PV...,8.0,40,['d7219f76b31818b7b13bf08dda81003d752e53f76608...
9,566248fe-36d9-4a5b-ac41-3e78c624fc7b,9,PHOTOVOLTAIK (PV),ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,The entire strategy is centered on the expansi...,10.0,43,['d7219f76b31818b7b13bf08dda81003d752e53f76608...


## 8. Relationship review table

This is the second main manual-review file.

Later we can classify relationships as:
- source-supported
- inferred
- questionable
- reject


In [31]:
view("05_relationship_review.csv", rows=30)


05_relationship_review.csv
Rows: 414
Columns: 14

Column names:
['id', 'human_readable_id', 'source', 'target', 'description', 'weight', 'combined_degree', 'text_unit_ids', 'source_raw_type', 'target_raw_type', 'evidence_status', 'suggested_relation_type', 'keep_in_controlled_kg', 'review_notes']



,id,human_readable_id,source,target,description,weight,combined_degree,text_unit_ids,source_raw_type,target_raw_type,evidence_status,suggested_relation_type,keep_in_controlled_kg,review_notes
0,ee487943-e36d-4146-8e2e-b7a9a1fd3067,0,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,"The BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT,...",19.0,48,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,ORGANIZATION,EVENT,NaN,NaN,NaN,NaN
1,e87db637-71f3-4134-8a02-954d1e518207,1,HUBERT FECHNER,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,Hubert Fechner is the principal author of the ...,8.0,41,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,PERSON,EVENT,NaN,NaN,NaN,NaN
2,316e1382-2ca7-477e-9d9f-c5f6bab0b3ba,2,LEONORE GEWESSLER,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",Leonore Gewessler is the Federal Minister lead...,10.0,11,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,PERSON,ORGANIZATION,NaN,NaN,NaN,NaN
3,86b0f8da-0a86-48c8-990d-8fbbe165249b,3,LEONORE GEWESSLER,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,"Leonore Gewessler represents, promotes, and ov...",9.0,41,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,PERSON,EVENT,NaN,NaN,NaN,NaN
4,c2727a60-8f2d-4b7f-9fd0-25cd463e97c7,4,ÖSTERREICH,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,ÖSTERREICH (Austria) is both the creator and p...,67.0,121,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,GEO,EVENT,NaN,NaN,NaN,NaN
5,48dadaa4-1765-4d04-bab8-87e9047bf629,5,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",HUBERT FECHNER,Hubert Fechner authored the strategy document ...,6.0,11,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,ORGANIZATION,PERSON,NaN,NaN,NaN,NaN
6,dbda7288-2916-410f-95d8-54283a4ce66a,6,WIEN,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",Vienna is the seat of the Ministry and the pub...,7.0,17,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,GEO,ORGANIZATION,NaN,NaN,NaN,NaN
7,34e20a8c-8899-486c-afa6-f1467d515bfb,7,ERNEUERBAREN-AUSBAU-GESETZ (EAG),ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,The EAG is a backbone legal act referenced and...,9.0,44,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,EVENT,EVENT,NaN,NaN,NaN,NaN
8,3a9025be-b2d8-4591-ba85-1fd25a4a58d2,8,INTEGRIERTER ÖSTERREICHISCHER NETZINFRASTRUKTU...,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,The strategy references and builds upon the PV...,8.0,40,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,EVENT,EVENT,NaN,NaN,NaN,NaN
9,566248fe-36d9-4a5b-ac41-3e78c624fc7b,9,PHOTOVOLTAIK (PV),ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,The entire strategy is centered on the expansi...,10.0,43,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,EVENT,EVENT,NaN,NaN,NaN,NaN


## 9. Communities


In [32]:
view("06_communities_all.csv", rows=30)


06_communities_all.csv
Rows: 62
Columns: 12

Column names:
['id', 'human_readable_id', 'community', 'level', 'parent', 'children', 'title', 'entity_ids', 'relationship_ids', 'text_unit_ids', 'period', 'size']



,id,human_readable_id,community,level,parent,children,title,entity_ids,relationship_ids,text_unit_ids,period,size
0,a438a991-b71d-469b-a1b5-20ef8ed2c8e5,0,0,0,-1,[16 17 18],Community 0,['c3251355-9488-4c60-a9ab-6be8c1c1c7ea'\n '164...,['163f4474-cccd-44af-aa9b-30d7c7f7d92b'\n '1e3...,['05802f69e17001a506260c74ca400f6a1368fbf01327...,2026-07-26,19
1,89a0b61a-0b65-430f-98f6-2580e7e8aca0,1,1,0,-1,[],Community 1,['13117ee5-b90c-4b76-8040-493a82f75bab'\n '7f2...,['15562353-e4ea-483e-af2b-2b6790c3c853'\n '2c7...,['41bd25afdc1cce74b36dd6db5e44e91354172fda1647...,2026-07-26,9
2,c01b5153-8328-46aa-a474-c9a5e37367c1,2,2,0,-1,[19 20 21 22],Community 2,['266cefa0-e9a7-467f-861d-edd83f4640b8'\n '215...,['0915d977-7c3a-4602-a1b3-e524e099b557'\n '0b4...,['23663988970639bea940978ccbd9150fd292856f2f31...,2026-07-26,20
3,bca47c41-c55b-43cb-8d78-2e9c04d3449a,3,3,0,-1,[23 24 25 26],Community 3,['54fa733a-a7c0-4381-a636-a9658fa0e87e'\n '8a8...,['077dc73a-13dd-4054-b2f1-5a225ae155d7'\n '0aa...,['23663988970639bea940978ccbd9150fd292856f2f31...,2026-07-26,12
4,063d9b6c-1392-43e8-8ff0-b709b5ca7f27,4,4,0,-1,[27 28],Community 4,['c6d4e772-b49e-41d5-9ea7-0f84b2a805ee'\n '4f4...,['0b1bd8a6-5def-4466-82c7-973c62019a02'\n '16a...,['05802f69e17001a506260c74ca400f6a1368fbf01327...,2026-07-26,12
5,97030300-0a79-4cc0-922a-7bded5ef6b8b,5,5,0,-1,[29 30 31 32 33 34 35],Community 5,['cb0a1b84-f9a3-4404-9807-afa3d1e6c3af'\n '586...,['02990fc8-a1a7-4ec9-9d12-ea19e65ee6ef'\n '0db...,['05802f69e17001a506260c74ca400f6a1368fbf01327...,2026-07-26,34
6,24ff87cd-f2b4-41ae-9537-e0e56ccbce3a,6,6,0,-1,[36 37 38 39],Community 6,['6f588fa0-b739-4da0-8b84-acfb7bf40dda'\n '00d...,['019059ec-b98f-4eb6-b9c0-1553ffa9ae88'\n '09c...,['4d221520fe7090409cfa9cda598db30683356c340ab3...,2026-07-26,14
7,df080313-8d00-40cc-b55f-af434832b77c,7,7,0,-1,[],Community 7,['5941618c-31ba-4dc7-8ec3-057ae490aabd'\n '41d...,['1eccd143-266e-469a-8809-fe231f112954'\n '200...,['50e06bf07ca2ce3a4e0dc0520305a36334e339dc022e...,2026-07-26,7
8,fee0f963-186d-4545-b6ef-1cfe13ada4ff,8,8,0,-1,[40 41 42 43],Community 8,['f8a2b891-80cf-4a82-9115-71f503f46d34'\n 'e60...,['042f5191-40c6-43f5-b37f-e24f7effada6'\n '112...,['23663988970639bea940978ccbd9150fd292856f2f31...,2026-07-26,19
9,43d79948-58b9-4f05-9fe0-f465bcba2ca5,9,9,0,-1,[44 45 46 47],Community 9,['28ad2239-3425-467c-a5b7-d2d823e3d34d'\n '7a0...,['04ee5f0f-2e1f-4ada-a715-7be714b884b7'\n '40d...,['05802f69e17001a506260c74ca400f6a1368fbf01327...,2026-07-26,12


## 10. Community reports


In [33]:
view("07_community_reports_all.csv", rows=20)


07_community_reports_all.csv
Rows: 62
Columns: 15

Column names:
['id', 'human_readable_id', 'community', 'level', 'parent', 'children', 'title', 'summary', 'full_content', 'rank', 'rating_explanation', 'findings', 'full_content_json', 'period', 'size']



,id,human_readable_id,community,level,parent,children,title,summary,full_content,rank,rating_explanation,findings,full_content_json,period,size
0,a13b29e2a420bd349c8f9109e6022a7eff9c597d717531...,60,60,2,29,[],Austrian Federal Government and National PV De...,This community centers on the Austrian Federal...,# Austrian Federal Government and National PV ...,8.0,The impact severity rating is high because the...,[{'explanation': 'The Austrian Federal Governm...,"{\n ""title"": ""Austrian Federal Government a...",2026-07-26,2
1,26230d3544d875be82557e5e8a2d80bb6e50c105db077b...,61,61,2,29,[],Austrian Photovoltaic Strategy Leadership and ...,This community centers around Austria's nation...,# Austrian Photovoltaic Strategy Leadership an...,8.5,The community commands high impact due to its ...,"[{'explanation': 'The ""Österreichische Photovo...","{\n ""title"": ""Austrian Photovoltaic Strateg...",2026-07-26,11
2,4cfc69a997bb9b15ad32bfa02086d886bfddcb1f76ce50...,16,16,1,0,[],E-Control and Austria's Grid Connection Action...,"This community centers on E-Control, Austria’s...",# E-Control and Austria's Grid Connection Acti...,8.5,The impact severity rating is high due to E-Co...,"[{'explanation': ""E-Control occupies a central...","{\n ""title"": ""E-Control and Austria's Grid ...",2026-07-26,6
3,003b89cc2b10773b1215b9f1c80d85aace49d6e3a4795f...,17,17,1,0,[],Austria’s Photovoltaic and Energy Transition S...,This community comprises key Austrian institut...,# Austria’s Photovoltaic and Energy Transition...,8.0,The impact severity rating is high due to the ...,[{'explanation': 'BMK (the Austrian Federal Mi...,"{\n ""title"": ""Austria’s Photovoltaic and En...",2026-07-26,9
4,6c92ae3c680eb80aff4de648ad303a3d98042d7efd5516...,18,18,1,0,[],Austrian Federal Ministry for Climate Action a...,This community centers around Austria's Federa...,# Austrian Federal Ministry for Climate Action...,8.5,The BMK community poses high impact due to its...,"[{'explanation': ""The Bundesministerium für Kl...","{\n ""title"": ""Austrian Federal Ministry for...",2026-07-26,4
5,e7194ecf611b10a2fb9ba5600ea1208399a1cbe30a57af...,19,19,1,2,[],Austrian Renewable Expansion Act and AGRI-PV-A...,This community centers on the implementation o...,# Austrian Renewable Expansion Act and AGRI-PV...,8.5,The community has a high impact severity ratin...,[{'explanation': 'The Austrian Renewable Expan...,"{\n ""title"": ""Austrian Renewable Expansion ...",2026-07-26,4
6,4517b31e6c628891cde69690d418e600c1c4db5297bac0...,20,20,1,2,[],Austria's Photovoltaic Sector and Energy Marke...,This community centers on Austria’s rapidly ev...,# Austria's Photovoltaic Sector and Energy Mar...,8.5,The severity rating is high due to the communi...,"[{'explanation': 'PV-ANLAGE installations, enc...","{\n ""title"": ""Austria's Photovoltaic Sector...",2026-07-26,12
7,ab32d2277b97f91d2d5732a8f957b8a7b550ef8f66b7e3...,21,21,1,2,[],Austrian Biodiversitäts-Solarpark and Freifläc...,This community encompasses Austria's effort to...,# Austrian Biodiversitäts-Solarpark and Freifl...,8.5,"The community’s integrated, dual-focus approac...",[{'explanation': 'Both FREIFLÄCHEN-BIODIVERSIT...,"{\n ""title"": ""Austrian Biodiversitäts-Solar...",2026-07-26,2
8,c470e0f78ce2e1b2aca5c845144f59ba24b5bfbee7b62a...,22,22,1,2,[],Austrian Solarindustrie and PV-Wertschöpfungsk...,The Austrian solar industry (Solarindustrie) a...,# Austrian Solarindustrie and PV-Wertschöpfung...,8.0,The impact severity rating is high due to the ...,"[{'explanation': 'The Austrian Solarindustrie,...","{\n ""title"": ""Austrian Solarindustrie and P...",2026-07-26,2
9,a25ffc1b527d0cb3252e8e7d6f0c1b4d541ce26aeb7107...,23,23,1,3,[],Austrian Unternehmen and the EU-Taxonomy Susta...,This community centers around Austrian Unterne...,# Austrian Unternehmen and the EU-Taxonomy Sus...,8.0,The community is rated highly impactful due to...,"[{'explanation': ""Austrian corporate entities ...","{\n ""title"": ""Austrian Unternehmen and the ...",2026-

## 11. Source documents


In [34]:
view("08_documents_all.csv", rows=20)


08_documents_all.csv
Rows: 1
Columns: 7

Column names:
['id', 'human_readable_id', 'title', 'text', 'text_unit_ids', 'creation_date', 'raw_data']



,id,human_readable_id,title,text,text_unit_ids,creation_date,raw_data
0,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,0,AT_STRATEGY_Austrian_Photovoltaic_Strategy_202...,Österreichische Photovoltaik-\nStrategie\n\nZi...,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,2026-07-26 22:20:50 +0200,NaN


## 12. Text units / chunks


In [35]:
view("09_text_units_all.csv", rows=20)


09_text_units_all.csv
Rows: 17
Columns: 8

Column names:
['id', 'human_readable_id', 'text', 'n_tokens', 'document_id', 'entity_ids', 'relationship_ids', 'covariate_ids']



,id,human_readable_id,text,n_tokens,document_id,entity_ids,relationship_ids,covariate_ids
0,d7219f76b31818b7b13bf08dda81003d752e53f7660831...,0,Österreichische Photovoltaik-\nStrategie\n\nZi...,1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['cb0a1b84-f9a3-4404-9807-afa3d1e6c3af'\n '822...,['ee487943-e36d-4146-8e2e-b7a9a1fd3067'\n 'e87...,[]
1,50fb8f6506c2271985ee7ffe42523d2eb06abbd85be8ea...,1,\n\n6.4.1 Maßnahmen ............................,1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['05c9af58-ac32-4ec6-b923-358b587e8cae'\n 'a90...,['0a7b5893-ad12-483f-81b2-a48bcabd624e'\n '5fd...,[]
2,50e06bf07ca2ce3a4e0dc0520305a36334e339dc022e32...,2,"Im Jahr 2023 waren 6,3 TWh er-\n\nreicht.\n\n...",1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['05c9af58-ac32-4ec6-b923-358b587e8cae'\n 'a90...,['1135fa09-924a-4fd4-ad10-3895fb2dee28'\n '71d...,[]
3,e498566a78d36b3ce13304157993b9b4269045840f1d63...,3,"energie, Wasserkraft, Wind- und\n\nBioenergie ...",1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['cb0a1b84-f9a3-4404-9807-afa3d1e6c3af'\n 'cb0...,['c2727a60-8f2d-4b7f-9fd0-25cd463e97c7'\n 'c27...,[]
4,f9e54ba797ff2b80a5f0e5f24e3901236ac4130e4e84d0...,4,ein Exportschlager der heimischen\n\nWirts...,1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['cb0a1b84-f9a3-4404-9807-afa3d1e6c3af'\n '05c...,['c2727a60-8f2d-4b7f-9fd0-25cd463e97c7'\n 'e30...,[]
5,06c273ec9919eb4693462fdacebd50536ccdcf47bacc47...,5,langfristige Wirtschaftlichkeit und die Marktk...,1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['cb0a1b84-f9a3-4404-9807-afa3d1e6c3af'\n '05c...,['aa4a8c35-ffc8-48fa-b27d-77ef5e571f99'\n '9a4...,[]
6,23663988970639bea940978ccbd9150fd292856f2f31a1...,6,"bevorzugte PV-An-\n\nwendungen, sowie eine Au...",1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['cb0a1b84-f9a3-4404-9807-afa3d1e6c3af'\n '37e...,['78386c17-ce06-4ac9-9b9a-fffa45df213e'\n 'b46...,[]
7,41bd25afdc1cce74b36dd6db5e44e91354172fda16475a...,7,ierungszie-\nlen der EU9. Im Bereich der Landw...,1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['05c9af58-ac32-4ec6-b923-358b587e8cae'\n 'a90...,['fc9dc79a-d096-49c9-b8c3-1e14e447409f'\n '816...,[]
8,6a1a5132f360531b1174e0eac2b2c5b6ece3b0771a8037...,8,sorgung). Trans-\n\nparente Qualitätsauszeichn...,1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['cb0a1b84-f9a3-4404-9807-afa3d1e6c3af'\n '05c...,['b45ce5a6-e552-4510-a31c-6bad6a3a93c0'\n 'a21...,[]
9,2d9fdebd364caf6ea226dc0ee6624e0c2d54fc65848a9c...,9,Ein Export der PV-Produktion zu erzeugungsstar...,1200,39ed356c790f07e222ffe8e84bfb78009537e51e126410...,['cb0a1b84-f9a3-4404-9807-afa3d1e6c3af'\n '05c...,['c2727a60-8f2d-4b7f-9fd0-25cd463e97c7'\n '119...,[]


## 13. Possible duplicate entities


In [36]:
view("10_possible_duplicates.csv", rows=50)


10_possible_duplicates.csv
Rows: 2
Columns: 6

Column names:
['title', 'normalized_title', 'type', 'description', 'frequency', 'degree']



,title,normalized_title,type,description,frequency,degree
0,"FTI-SCHWERPUNKT ""ENERGIEWENDE""",FTI SCHWERPUNKT ENERGIEWENDE,EVENT,"The FTI-Schwerpunkt ""Energiewende"" is a nation...",1,1
1,FTI-SCHWERPUNKT ENERGIEWENDE,FTI SCHWERPUNKT ENERGIEWENDE,EVENT,"The FTI-Schwerpunkt ""Energiewende"" is a resear...",1,2


## 14. Top connected entities


In [37]:
view("11_top_connected_entities.csv", rows=50)


11_top_connected_entities.csv
Rows: 30
Columns: 5

Column names:
['title', 'type', 'degree', 'frequency', 'description']



,title,type,degree,frequency,description
0,ÖSTERREICH,GEO,82,15,"Österreich, also known as Austria, is a Centra..."
1,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,EVENT,39,14,"The ""Österreichische Photovoltaik-Strategie"" (..."
2,PV-ANLAGE,ORGANIZATION,20,3,PV-ANLAGE (Photovoltaic system) refers to dece...
3,BMK,ORGANIZATION,16,7,"BMK, the commonly used abbreviation for the Au..."
4,BUND,ORGANIZATION,14,1,"The federal government of Austria, which colla..."
5,KLIMA- UND ENERGIEFONDS,ORGANIZATION,14,6,The **KLIMA- UND ENERGIEFONDS** (Climate and E...
6,ENERGIEGEMEINSCHAFTEN,ORGANIZATION,14,6,ENERGIEGEMEINSCHAFTEN (energy communities) are...
7,PHOTOVOLTAIKANLAGEN,ORGANIZATION,13,2,Photovoltaikanlagen (Photovoltaic Installation...
8,E-CONTROL,ORGANIZATION,11,6,E-CONTROL is the official Austrian regulatory ...
9,BUNDESLÄNDER,GEO,11,4,"The ""Bundesländer"" are the federal states with..."


## 15. Top relationships


In [38]:
view("12_top_relationships.csv", rows=50)


12_top_relationships.csv
Rows: 50
Columns: 4

Column names:
['source', 'target', 'description', 'weight']



,source,target,description,weight
0,ÖSTERREICH,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,ÖSTERREICH (Austria) is both the creator and p...,67.0
1,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,ÖSTERREICH,"The ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE, al...",34.0
2,ÖSTERREICH,BUNDESLÄNDER,"ÖSTERREICH, also known as Austria, is divided ...",33.0
3,BMK,ÖSTERREICH,"BMK, or the Federal Ministry for Climate Actio...",27.0
4,ENERGIEWENDE,ÖSTERREICH,The Energiewende is a major strategic initiati...,19.0
5,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,"The BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT,...",19.0
6,ÖSTERREICH,KLIMANEUTRALITÄT 2040,Austria (ÖSTERREICH) has established an ambiti...,19.0
7,EUROPÄISCHE UNION,ÖSTERREICH,Austria (ÖSTERREICH) is a member state of the ...,18.0
8,E-CONTROL,ÖSTERREICH,E-Control is Austria’s national regulatory aut...,18.0
9,ÖSTERREICH,EU,"ÖSTERREICH, or Austria, is a member state of t...",17.0


## 16. Community report review


In [39]:
view("13_community_report_review.csv", rows=30)


13_community_report_review.csv
Rows: 62
Columns: 18

Column names:
['id', 'human_readable_id', 'community', 'level', 'parent', 'children', 'title', 'summary', 'full_content', 'rank', 'rating_explanation', 'findings', 'full_content_json', 'period', 'size', 'source_supported', 'possible_overgeneralization', 'review_notes']



,id,human_readable_id,community,level,parent,children,title,summary,full_content,rank,rating_explanation,findings,full_content_json,period,size,source_supported,possible_overgeneralization,review_notes
0,a13b29e2a420bd349c8f9109e6022a7eff9c597d717531...,60,60,2,29,[],Austrian Federal Government and National PV De...,This community centers on the Austrian Federal...,# Austrian Federal Government and National PV ...,8.0,The impact severity rating is high because the...,[{'explanation': 'The Austrian Federal Governm...,"{\n ""title"": ""Austrian Federal Government a...",2026-07-26,2,NaN,NaN,NaN
1,26230d3544d875be82557e5e8a2d80bb6e50c105db077b...,61,61,2,29,[],Austrian Photovoltaic Strategy Leadership and ...,This community centers around Austria's nation...,# Austrian Photovoltaic Strategy Leadership an...,8.5,The community commands high impact due to its ...,"[{'explanation': 'The ""Österreichische Photovo...","{\n ""title"": ""Austrian Photovoltaic Strateg...",2026-07-26,11,NaN,NaN,NaN
2,4cfc69a997bb9b15ad32bfa02086d886bfddcb1f76ce50...,16,16,1,0,[],E-Control and Austria's Grid Connection Action...,"This community centers on E-Control, Austria’s...",# E-Control and Austria's Grid Connection Acti...,8.5,The impact severity rating is high due to E-Co...,"[{'explanation': ""E-Control occupies a central...","{\n ""title"": ""E-Control and Austria's Grid ...",2026-07-26,6,NaN,NaN,NaN
3,003b89cc2b10773b1215b9f1c80d85aace49d6e3a4795f...,17,17,1,0,[],Austria’s Photovoltaic and Energy Transition S...,This community comprises key Austrian institut...,# Austria’s Photovoltaic and Energy Transition...,8.0,The impact severity rating is high due to the ...,[{'explanation': 'BMK (the Austrian Federal Mi...,"{\n ""title"": ""Austria’s Photovoltaic and En...",2026-07-26,9,NaN,NaN,NaN
4,6c92ae3c680eb80aff4de648ad303a3d98042d7efd5516...,18,18,1,0,[],Austrian Federal Ministry for Climate Action a...,This community centers around Austria's Federa...,# Austrian Federal Ministry for Climate Action...,8.5,The BMK community poses high impact due to its...,"[{'explanation': ""The Bundesministerium für Kl...","{\n ""title"": ""Austrian Federal Ministry for...",2026-07-26,4,NaN,NaN,NaN
5,e7194ecf611b10a2fb9ba5600ea1208399a1cbe30a57af...,19,19,1,2,[],Austrian Renewable Expansion Act and AGRI-PV-A...,This community centers on the implementation o...,# Austrian Renewable Expansion Act and AGRI-PV...,8.5,The community has a high impact severity ratin...,[{'explanation': 'The Austrian Renewable Expan...,"{\n ""title"": ""Austrian Renewable Expansion ...",2026-07-26,4,NaN,NaN,NaN
6,4517b31e6c628891cde69690d418e600c1c4db5297bac0...,20,20,1,2,[],Austria's Photovoltaic Sector and Energy Marke...,This community centers on Austria’s rapidly ev...,# Austria's Photovoltaic Sector and Energy Mar...,8.5,The severity rating is high due to the communi...,"[{'explanation': 'PV-ANLAGE installations, enc...","{\n ""title"": ""Austria's Photovoltaic Sector...",2026-07-26,12,NaN,NaN,NaN
7,ab32d2277b97f91d2d5732a8f957b8a7b550ef8f66b7e3...,21,21,1,2,[],Austrian Biodiversitäts-Solarpark and Freifläc...,This community encompasses Austria's effort to...,# Austrian Biodiversitäts-Solarpark and Freifl...,8.5,"The community’s integrated, dual-focus approac...",[{'explanation': 'Both FREIFLÄCHEN-BIODIVERSIT...,"{\n ""title"": ""Austrian Biodiversitäts-Solar...",2026-07-26,2,NaN,NaN,NaN
8,c470e0f78ce2e1b2aca5c845144f59ba24b5bfbee7b62a...,22,22,1,2,[],Austrian Solarindustrie and PV-Wertschöpfungsk...,The Austrian solar industry (Solarindustrie) a...,# Austrian Solarindustrie and PV-Wertschöpfung...,8.0,The impact severity rating is high due to the ...,"[{'explanation': 'The Austrian Solarindustrie,...","{\n ""title"": ""Austrian Solarindustrie and P...",2026-07-26,2,NaN,NaN,NaN
9,a25ffc1b527d0cb3252e8e7d6f0c1b4d541ce26aeb7107...,23,23,1,3,[],Austrian Unternehmen and the EU-Taxonomy Susta...,This community centers around Austrian Unterne...,# Austrian Unternehmen and the EU-Taxonomy Sus..

## 17. Search entities by word

Change `search_term` to anything you want, for example:
- `photovoltaik`
- `eag`
- `österreich`
- `netz`
- `förder`


In [40]:
search_term = "photovoltaik"

df = datasets["03_entity_review.csv"]

mask = df.astype(str).apply(
    lambda col: col.str.contains(search_term, case=False, na=False)
).any(axis=1)

display(df[mask])


,human_readable_id,title,type,description,frequency,degree,text_unit_ids,suggested_schema_type,canonical_name,keep_in_controlled_kg,record_linkage_group,review_notes
0,0,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,EVENT,"The ""Österreichische Photovoltaik-Strategie"" (...",14,39,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
8,8,PHOTOVOLTAIK (PV),EVENT,The technological process of converting solar ...,1,4,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
9,9,CAJETAN PERWEIN,PERSON,Cajetan Perwein is credited as a photographer ...,1,1,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,NaN,NaN,NaN,NaN,NaN
26,26,PHOTOVOLTAIK-STRATEGIE,EVENT,Die österreichische Photovoltaik-Strategie ist...,1,6,['50e06bf07ca2ce3a4e0dc0520305a36334e339dc022e...,NaN,NaN,NaN,NaN,NaN
36,36,PHOTOVOLTAIK-AUSBAU,EVENT,Der Ausbau der Photovoltaik in Österreich ist ...,1,2,['50e06bf07ca2ce3a4e0dc0520305a36334e339dc022e...,NaN,NaN,NaN,NaN,NaN
56,56,PHOTOVOLTAIKANLAGEN,ORGANIZATION,Photovoltaikanlagen (Photovoltaic Installation...,2,13,['f9e54ba797ff2b80a5f0e5f24e3901236ac4130e4e84...,NaN,NaN,NaN,NaN,NaN
66,66,PHOTOVOLTAIK,EVENT,PHOTOVOLTAIK refers to the deployment and expa...,2,5,['06c273ec9919eb4693462fdacebd50536ccdcf47bacc...,NaN,NaN,NaN,NaN,NaN
121,121,MUSTER- & LEUCHTTURMPROJEKTE PHOTOVOLTAIK,EVENT,The Muster- & Leuchtturmprojekte Photovoltaik ...,1,1,['6a1a5132f360531b1174e0eac2b2c5b6ece3b0771a80...,NaN,NaN,NaN,NaN,NaN
161,161,PHOTOVOLTAIK-PROJEKTE,EVENT,Photovoltaic projects are solar power installa...,1,4,['4d221520fe7090409cfa9cda598db30683356c340ab3...,NaN,NaN,NaN,NaN,NaN


## 18. Search relationships by word

Change `search_term` to investigate a topic.


In [41]:
search_term = "photovoltaik"

df = datasets["05_relationship_review.csv"]

mask = df.astype(str).apply(
    lambda col: col.str.contains(search_term, case=False, na=False)
).any(axis=1)

display(df[mask])


,id,human_readable_id,source,target,description,weight,combined_degree,text_unit_ids,source_raw_type,target_raw_type,evidence_status,suggested_relation_type,keep_in_controlled_kg,review_notes
0,ee487943-e36d-4146-8e2e-b7a9a1fd3067,0,"BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT, ENE...",ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,"The BUNDESMINISTERIUM FÜR KLIMASCHUTZ, UMWELT,...",19.0,48,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,ORGANIZATION,EVENT,NaN,NaN,NaN,NaN
1,e87db637-71f3-4134-8a02-954d1e518207,1,HUBERT FECHNER,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,Hubert Fechner is the principal author of the ...,8.0,41,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,PERSON,EVENT,NaN,NaN,NaN,NaN
3,86b0f8da-0a86-48c8-990d-8fbbe165249b,3,LEONORE GEWESSLER,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,"Leonore Gewessler represents, promotes, and ov...",9.0,41,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,PERSON,EVENT,NaN,NaN,NaN,NaN
4,c2727a60-8f2d-4b7f-9fd0-25cd463e97c7,4,ÖSTERREICH,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,ÖSTERREICH (Austria) is both the creator and p...,67.0,121,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,GEO,EVENT,NaN,NaN,NaN,NaN
7,34e20a8c-8899-486c-afa6-f1467d515bfb,7,ERNEUERBAREN-AUSBAU-GESETZ (EAG),ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,The EAG is a backbone legal act referenced and...,9.0,44,['d7219f76b31818b7b13bf08dda81003d752e53f76608...,EVENT,EVENT,NaN,NaN,NaN,NaN
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
345,c618bc16-f9d2-489c-a04e-cae770590528,345,FIT FÜR 55,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,Austria’s photovoltaic policy aligns with the ...,6.0,43,['05802f69e17001a506260c74ca400f6a1368fbf01327...,EVENT,EVENT,NaN,NaN,NaN,NaN
374,b3361593-90d6-4465-9247-addaee363a9b,374,BUNDESVERBAND PV AUSTRIA,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,Bundesverband PV Austria is involved in shapin...,6.0,46,['6927535991c09031efd0df39ec21591dd3bb460da14c...,ORGANIZATION,EVENT,NaN,NaN,NaN,NaN
400,ff337f7e-4657-428a-b4d7-d19e98663e83,400,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,PV-ANLAGEN,The PV strategy is developed to advance the in...,9.0,46,['6927535991c09031efd0df39ec21591dd3bb460da14c...,EVENT,EVENT,NaN,NaN,NaN,NaN
405,82d1a5db-4db4-4a29-b7ba-73ad1004cb75,405,UMWELTVERTRÄGLICHKEITSPRÜFUNG,ÖSTERREICHISCHE PHOTOVOLTAIK-STRATEGIE,Environmental impact assessments are a relevan...,6.0,40,['7ae981575e185e2452d144d541d822b7725e27bc7dda...,EVENT,EVENT,NaN,NaN,NaN,NaN


## 19. Show a full dataset

Jupyter normally truncates large tables.  
Use the line below for a full dataset when needed.

Be careful: entities have 312 rows and relationships have 414 rows.


In [42]:
# Example:
# display(datasets["03_entity_review.csv"])

# Uncomment the line below when you want the complete entity review table:
# display(datasets["03_entity_review.csv"])


## Recommended inspection order

1. `02_entity_type_counts.csv`
2. `03_entity_review.csv`
3. `10_possible_duplicates.csv`
4. `11_top_connected_entities.csv`
5. `05_relationship_review.csv`
6. `12_top_relationships.csv`
7. `07_community_reports_all.csv`
8. `09_text_units_all.csv`

We will compare the raw GraphRAG output with your Solar Market Schema V1 rather than accepting the raw graph automatically.
